Correct invalid groups_attribute_path option + added details for nested groups (#61202)
grafana/grafana: docs/sources/setup-grafana/configure-security/configure-authentication/keycloak/index.md

```ini
[auth.generic_oauth]
groups_attribute_path = groups
```

> If using nested groups that contain special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana will properly evaluate nested groups. eg. a parent group named `Global` with nested group `department` containing a list of groups:

Reviewer: ```suggestion
If you use nested groups containing special characters such as quotes or colons, the JMESPath parser can perform a harmless reverse function so Grafana can properly evaluate nested groups. For example, a parent group named `Global` with nested group `department` containing a list of groups:
```

Reviewer: @IevaVasiljeva , can you please take a look at this PR? If not you, who can help? Thank you!
